android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -98,6 +98,11 @@ public int getAppVersion() {
     }
 
     @NonNull
+    public String getAppVersionName() {
+        return applicationContext.getResources().getString(R.string.osb_app_version_name);
```

Comment:
If the header is "StudyName/X (Nexus 6P; Android 23) BridgeSDK/Y", then versionName is recommended to be "version A.B.C, build X"

Does that sound right?